src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/Collector{T}.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public class Collector<T> : ICollector<T>
```

Comment:
does this need to be public? maybe put in the internals abstraction?

Reply:
Good catch, agreed.